### PR TITLE
refs #11190 - add instructions for installer from git

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,17 @@ Install without RVM
 katello-devel-installer --katello-use-rvm false
 ```
 
+### From Git
+
+If you're working off the master branch, use Librarian to pull in all the modules to your local checkout:
```
Comment (ehelms) [inline README.md]: Could we include installation of librarian-puppet and puppet ruby > 2.0 since that isn't obvious?

Comment (stbenjam) [inline README.md]: There's a Gemfile now

Comment (stbenjam) [inline README.md]: Is that enough? Or should I specifically mention the requirements?

**Comment by ehelms** [comment]
Oh, good call, LGTM! Could you reference the removal issue # so we aren't breaking protocol too much?

**Comment by stbenjam** [comment]
Updated

**Comment by ehelms** [comment]
ACK

**Comment by ehelms** [comment]
ACK

**Pull request merged**